### DISCUSSION TITLE
Codespace is 404ing

### DISCUSSION THREAD

**Discussion by pamelafox** [community/community#54393] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

Earlier this week, I created a Codespace on a branch in this fork:
https://github.com/pamelafox/awesome-azd/tree/main

I just updated that repo to match the upstream (after my branch was merged), and attempted to go back into the Codespace. The tab says that Codespace is still active and is on 'main' branch:

![Screenshot 2023-05-03 at 3 19 21 PM](https://user-images.githubusercontent.com/297042/236062849-89b75645-e177-446f-acb3-02e8e5e00dcc.png)

However, that URL will no longer load for me:

![Screenshot 2023-05-03 at 3 23 52 PM](https://user-images.githubusercontent.com/297042/236062991-a2c7a0cd-8c55-43a4-9d70-3f0a6f459db5.png)

**Comment by pamelafox** [comment]
Update: It's also not working if I press '+' inside the Codespace tab to attempt to get a new Codespace:
https://pamelafox-obscure-space-invention-4wpr7g9552qj4g.github.dev/

I assume it'll work if I delete the Codespace but I'll leave it running for your debugging purposes.

**Comment by dmgardiner25** [comment, marked as answer]
@pamelafox thanks for reporting and sorry about that! We had a brief incident where some users were unable to connect to their codespace but that should be resolved now. Please let me know if you are still having issues.

**Comment by pamelafox** [comment]
Ah it does seem to be working now, thanks!
